src/EnergyPlus/CMakeLists.txt

```diff
@@ -612,3 +612,49 @@ if(UNIX AND NOT APPLE)
   COMMAND ${CMAKE_COMMAND} -E create_symlink energyplus EnergyPlus
   COMMAND ${CMAKE_COMMAND} -E rename EnergyPlus "${CMAKE_BINARY_DIR}/Products/EnergyPlus") 
 endif()
+
+# if we are building a package, we can do additional operations, such as run doc scripts
+if(BUILD_PACKAGE)
+  
+  # Some docs are generated on the fly here, create a dir for the 'built' files
+  set( DOCS_OUT "${CMAKE_BINARY_DIR}/autodocs" )
+  file( MAKE_DIRECTORY "${DOCS_OUT}" )
+  
+  # the output variables listing
+  add_custom_command( 
+    TARGET energyplus
+    PRE_BUILD
+    COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/doc/tools/parse_output_variables.py" "${CMAKE_SOURCE_DIR}/src/EnergyPlus" "${DOCS_OUT}/SetupOutputVariables.csv" "${DOCS_OUT}/SetupOutputVariables.md" 
+  )
+  install(FILES "${CMAKE_BINARY_DIR}/autodocs/SetupOutputVariables.csv" DESTINATION "./")
+  install(FILES "${CMAKE_BINARY_DIR}/autodocs/SetupOutputVariables.md" DESTINATION "./")
+  
+  # the example file summary
+  add_custom_command(
+    TARGET energyplus
+    PRE_BUILD
+    COMMAND "${PYTHON_EXECUTABLE}" "${CMAKE_SOURCE_DIR}/doc/tools/example_file_summary.py" "${CMAKE_SOURCE_DIR}/testfiles" "${DOCS_OUT}/ExampleFiles.html"
```

Comment:
Why does this output html directly while other products are md?  It seems inconsistent, however I think I prefer the html as the finished product.  Might we convert those md files to html and install that?  Of course we don't yet have a dependency to do that.